core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctLong256GroupByFunction.java

```diff
@@ -115,34 +119,74 @@ public boolean isConstant() {
 
     @Override
     public boolean isParallelismSupported() {
-        return false;
+        return true;
```

Comment:
I'm changing these methods in other functions to a more correct implementation in another PR, so let's change it to:
```suggestion
        return UnaryFunction.super.isParallelismSupported();
```